components/Modal/CreateImage.js

```diff
@@ -96,6 +96,7 @@ class CreateImage extends React.Component {
       <div
         className="modal fade"
         id="cmpsr-modal-crt-image"
+        data-id="cmpsr-modal-crt-image"
```

Comment:
We don't need the extra `data-id` attribute on this element. Since we already have an `id` attribute here, we can use that instead of create a new `data-id` one.

For example, the id that's defined here is currently being used in [test/end-to-end/pages/createImage.js, line 12](https://github.com/weldr/welder-web/pull/375/files#diff-886584fb195b433a216cecc7e230cd8cR12) as `#cmpsr-modal-crt-image`

Reply:
Are `id` and `data-id` pretty much the same in this instance? If an element already has an `id` attribute does that mean nothing needs to be changed?

Reply:
Correct!